truss/templates/server.Dockerfile.jinja

```diff
@@ -106,7 +108,7 @@ RUN mkdir -p {{ supervisor_log_dir }}
 COPY supervisord.conf {{ supervisor_config_path }}
 ENV SUPERVISOR_SERVER_URL="{{ supervisor_server_url }}"
 ENV SERVER_START_CMD="supervisord -c {{ supervisor_config_path }}"
-ENTRYPOINT ["/usr/local/bin/supervisord", "-c", "{{ supervisor_config_path }}"]
+ENTRYPOINT ["supervisord", "-c", "{{ supervisor_config_path }}"]
```

Comment:
one non-blocking comment, since this resulted in an actual bug/issue, would be awesome if we could also have an integration test for this, don't know how easy that would be to put together

Reply:
No easy way on top of my mind, maybe what we want is to have a different testing docker image which installs supervisord in a different path? Any better ideas?

Reply:
yeah maybe something like that. no need to block on this then